modules/ossm-migrating-workloads-using-the-istio-revision-label.adoc

```diff
@@ -0,0 +1,116 @@
+// Module included in the following assemblies:
+//
+// * service-mesh-docs-main/migrating/checklists/ossm-migrating-cluster-wide-assembly.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ossm-migrating-workloads-using-the-istio-revision-label_{context}"]
+= Migrating workloads using the Istio revision label
+
+Now you can migrate your workloads from the {SMProduct} 2.6 control plane to the {SMproduct} 3.0 control plane.
+
+Revision tags are not used in this example for simplicity. When migrating large meshes, you can use revision tags to avoid re-labeling all namespaces during future 3.y.z upgrades.
+
+[NOTE]
+====
+You can migrate workloads and gateways separately, and in any order. For more information, see "Migrating gateways".
+====
+
+.Procedure
+
+. Find the current `IstioRevision` for your {SMProduct} 3.0 control plane by running the following command:
++
+[source,terminal]
+----
+$ oc get istios istio-tenant-a
+----
++
+.Example output
++
+[source,terminal]
+----
+NAME             REVISIONS   READY   IN USE   ACTIVE REVISION   STATUS    VERSION   AGE
+istio-tenant-a   1           1       0        istio-tenant-a    Healthy   v1.24.3   30s
```

Comment:
```suggestion
ossm-3-v1-24-3   1           1       0        ossm-3-v1-24-3    Healthy   v1.24.3   30s
```

Reply:
I will check

Reply:
You are right. I was too fast with manual edits without testing.

Reply:
done